### DISCUSSION TITLE
Regarding Access Control

### DISCUSSION THREAD

**Discussion by amd-yan** [executablebooks/meta#135] [General] [answered]
Thank you for creating the Jupyter Book. I just happen to see it online, it looks awesome! I have a quick question regarding access control. I checked all of the issues and discussions, but I didn't see one, so I decide to post my question here. I wonder if there is any way to control access to the Jupyter Book (either via password protection, integration with single sign-on, or wrapped by another web framework, eg: Django) once it is deployed to the server?

**Comment by choldgraf** [comment, marked as answer]
ah interesting - there's nothing out-of-the-box for this right now, but if there were the proper javascript library to control this kind of thing, I bet it wouldn't be too hard to set up something like this for a Jupyter Book

**Reply by amd-yan** [reply]
I will do some exploration on that then. Thanks for your prompt response!
